app/src/main/java/org/greenstand/android/TreeTracker/ui/Color.kt

```diff
@@ -0,0 +1,24 @@
+package org.greenstand.android.TreeTracker.ui
+
+import androidx.compose.ui.graphics.Color
+
+val colorPrimary = Color(0xFF75B926)
```

Comment:
There is already a colors file

Reply:
Yeah I think I did this before the latest changes in master. Will fix this as part of the updates